docs/docs/rest-api/public/api/v2/types/port.raml

```diff
@@ -0,0 +1,4 @@
+#%RAML 1.0 DataType
+type: integer
+minimum: 1
```

Comment:
hostport needs to have a value of 0 (or some other way to indicate that marathon should "just pick one" from the offered port range)